aws/karpenter
# test: add beta scale test

Fixes #N/A 

**Description**
This adds in v1beta1 Scale testing and adds in a alpha-e2e-scale trigger that would trigger the Alpha scale tests. 
**How was this change tested?**
- make verify
- make e2etests

**Does this change impact docs?**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
